### PR TITLE
Locking down class names

### PR DESCRIPTION
**Pull request by donato**
Internal structure is now in line with : https://github.com/jwplayer/jwplayer-commercial/wiki/Html-Structure-and-Classes
[#88947454]

### CODE REVIEW

File: src/js/view/view.js [hunk 1]
```diff
@@ -249,11 +249,21 @@ define([
             }
 
             _container = _playerElement.getElementsByClassName('jw-main')[0];
```
Comment (robwalch) [inline src/js/view/view.js]: @donato was `jw-main` removed? If so we need to create another container for plugins and a sweep needs to be done to cleanup orphaned code.

Comment (jw-kaurand) [inline src/js/view/view.js]: Seems like this is causing some issues with jwplayer-commercial/test/vast-linear.html due to it looking for the jw-main class

Comment (donato) [inline src/js/view/view.js]: Plugins will go into .jw-overlays, (this is annotated in lucidcharts using
a comment)

Comment (robwalch) [inline src/js/view/view.js]: ok. updated chart and commit. it read 'jw-overlay' which is taken by tooltips

**Pull request closed without merging**